lib/wasix/src/syscalls/wasix/fd_pipe.rs

```diff
@@ -1,8 +1,12 @@
+use std::sync::atomic::AtomicUsize;
+
 use virtual_fs::Pipe;
 
 use super::*;
 use crate::syscalls::*;
 
+static PIPE_NUMBER: AtomicUsize = AtomicUsize::new(0);
```

Comment:
docs for why this is needed and what's going on here.

Reply:
Shouldn't this be a counter somewhere on the env data?

Reply:
Good point! We use a hash of the file name to calculate inode numbers, so all pipes were getting the same. This serves to make pipe names unique. It doesn't really matter otherwise. Will add comments.